graylog2-web-interface/src/components/events/events/Events.jsx

```diff
@@ -238,10 +238,20 @@ class Events extends React.Component {
       return renderEmptyContent();
     }
 
-    const filter = parameters.filter.alerts;
+    const { query, filter: { alerts: filter } } = parameters;
     const excludedFile = filter === 'exclude' ? 'Events' : 'Alerts & Events';
     const entity = (filter === 'only' ? 'Alerts' : excludedFile);
 
+    const emptyListComponent = query ? (
+      <Alert bsStyle="info">
+        <Icon name="info-circle" />&nbsp;No {entity} found for the current search criteria.
+      </Alert>
+    ) : (
+      <Alert bsStyle="info">
+        <Icon name="info-circle" />&nbsp;No {entity} exist.
+      </Alert>
+    );
```

Comment:
Let's use shared components for this, just to be able to adjust the layout for all similar cases in one place.
We already have the `EmptyResult` component, maybe we can rename it to something like `NoSearchResult` and create another component like `NoEntitiesExist`?

Reply:
sounds good to me. I'll work on getting that updated

Reply:
@linuspahl this should be complete now. I created a new component called `NoEntitiesExist` but I did notice there is the `EmptyEntity` component which has other usages that explain what the entity is and a link to create a new one. I chose not to unify the two but if you would like me to use `EmptyEntity` for all cases then I will probably need some texts for the different entities to explain what they are. Let me know what you would like to do.

Reply:
Sounds good!